build.teamcity.ps1

```diff
@@ -0,0 +1,72 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+$ErrorActionPreference = 'Stop'
+
+& "$PSScriptRoot/Initialize-PowershellForDevelopment.ps1"
+
+$teamcityParameters = Get-TeamCityParameters
+Write-Host
+Write-Host "$($teamcityParameters.Count) TeamCity parameters found."
+Write-Host
+
+# Build and Test
+$params = @{
```

Comment:
Probably a separate ticket, but have we reached a point where a configuration object makes more sense than just continuing to add more switches?  Particularly if packaging ends up collapsing in.